recipe/meta.yaml

```
@@ -32,8 +32,11 @@ requirements:
 
 test:
   commands:
-    - test -f $PREFIX/lib/libminizip${SHLIB_EXT}  # [not win]
-    - if not exist %LIBRARY_LIB%\\libminizip.lib exit 1  # [win]
+    # presence of shared libraries
+    - test -f $PREFIX/lib/libminizip${SHLIB_EXT}        # [unix]
+    - if not exist %LIBRARY_BIN%\libminizip.dll exit 1  # [win]
+    - if not exist %LIBRARY_LIB%\libminizip.lib exit 1  # [win]
+
```

For unix you can add the test for the absence of static libs as well:
```suggestion

        # absence of static libraries
        - test ! -f $PREFIX/lib/libminizip.a                # [unix]
```